doc/sphinx/source/utils/RTW/add_a_recipe.rst

```diff
@@ -26,48 +26,80 @@ How to add a recipe to the |RTW|
+   alphabetical order for easy user reference:
+   :ref:`Currently tested recipes <currently_tested_recipes>`.
+
+#. ``<site>-recipes.cylc`` contains **two lists of dictionaries**. The lists
+   are ``FAST_RECIPES`` and ``MEDIUM_RECIPES``.
 
    .. hint::
-      The ``fast`` key in the example task definition above
-      (``[[process<fast=recipe_albedolandcover>]]``) should match name of the
-      option the recipe was added to in the ``[task parameters]`` section in
-      the ``esmvaltool/utils/recipe_test_workflow/flow.cylc`` file
+      ``FAST_RECIPES`` take *less* than 10 minutes to run at your site.
+      ``MEDIUM_RECIPES`` take *more* than 10 minutes.
+
+#. Add the recipe to one of lists as as a dictionary of ``key, value`` pairs.
+   E.g.::
+
+      {
+         'recipe_path': 'recipe_a_fast_recipe',
+         'actual': '1m30s, 1.5 GB on 2025-01-01',
+         'max_time': 'PT2M',
+         'max_memory': '2G',
+      }
+
+   Add the following information for each key:
+
+   .. list-table::
+      :widths: 25 75
+      :header-rows: 1
+
+      * - Key
+        - Value
+      * - ``recipe_path``
+        - The path to the recipe. Recipe paths are specified relative to
+          ``esmvaltool/recipes``. For recipes in subdirectories, ``--`` stands
+          for ``/`` since the latter is an illegal char.
+      * - ``actual``
+        - A note of  the recipe's actual resource usage. From the successful run
+          of the recipe on the compute server at your site.
+      * - ``max_time``
+        - Set the maximum amount of time the recipe has to complete. Use the
+          ISO8601 duration format (see `Cylc ISO8601 Durations`_ for more).
+      * - ``max_memory``
+        - Set the memory to allocate to running the recipe. Default units are
+          megabytes. Different units can be specified using the
+          suffix [K|M|G|T] (see `Slurm sbatch --mem`_ for more).
 
    .. hint::
-      Set the ``execution time limit`` to 10-20% more than the actual duration.
-      For actual durations of up to ``1m45s``, set the ``execution time limit``
-      to ``PT2M`` (2 minutes).
+      Set the ``max_time`` to 10-20% more than the actual duration. For actual
+      durations of up to ``1m45s``, set ``max_time`` to ``PT2M`` (2 minutes).
 
    .. hint::
       Try not to regularly waste more than 500 MiB in memory usage. Typically,
       rounding the actual memory usage up to the nearest integer is acceptable.
 
+#. Once the recipe dictionary is added to either ``FAST_RECIPES`` or
+   ``MEDIUM_RECIPES``, the recipe will run as part of the |RTW| at your site.
+   Next, the recipe will need |KGOs| to run successfully in the RTW.
+
+   .. _jinja2_templating_language:
+
+   .. note::
+      The ``<site>-recipes.cylc`` file is actually written in the `Jinja2`_
+      templating language. Jinja2 gives |Cylc| many powerful features (visit
+      `Cylc Jinja2`_). This is beyond the scope of this guide. Follow the links
+      for more information.
```

Comment:
I feel the opening points of this documentation should be updated to give more information about what will happen when adding a recipe. For example, they will be editing a single file in the RTW, and it's Jinja2, and then include this information.